src/cli/NodeIO.ts

```diff
@@ -9,12 +9,38 @@ import { messageError, JSONSchemaStore, JSONSchema, parseJSON } from "../quickty
 const isURL = require("is-url");
 const fetch = require("node-fetch");
 
-export async function readableFromFileOrURL(fileOrURL: string): Promise<Readable> {
+interface HttpHeaders {
+    [key: string]: string;
+}
+
+function parseHeaders(httpHeaders?: string[]): HttpHeaders {
+    if (!Array.isArray(httpHeaders)) {
+        return {};
+    }
+
+    return httpHeaders.reduce(
+        function (result: HttpHeaders, httpHeader: string) {
+            if (httpHeader !== undefined && httpHeader.length > 0) {
+                const [key, value] = httpHeader.split(":");
```

Comment:
This will not work if the input string is, say `Header-With-Colons: Foo:Bar:Baz`.  Maybe that's not allowed?

Reply:
Yes you're right, this is a bit naive. I'll take another crack at making it more robust.

Reply:
Updated, let me know what you think @schani.

Reply:
Looks good!